src/utilities/extendSchema.js

```diff
@@ -55,6 +55,16 @@ type Options = {|
    * Default: false
    */
   commentDescriptions?: boolean,
+
+  /**
+   * If provided, the schema will consider fields or types with names included
+   * in this list valid, even if they do not adhere to the specification's
+   * schema validation rules.
+   *
+   * This option is provided to ease adoption and may be removed in a future
+   * major release.
+   */
+  allowedLegacyNames?: ?Array<string>,
```

Comment:
Should this be added here? If so, should the result of `extendSchema` include both the allowed legacy names of the input schema in addition to these?